doc/dap.txt

```diff
@@ -580,6 +580,12 @@ The configuration values are set via `dap.defaults.fallback` (for global) or
                        See |'switchbuf'|. Defaults to the global `'switchbuf'`
                        setting.
 
+                       nvim-dap provides an additional `usevisible` option
+                       that can be used to prevent jumps within the active
+                       window if a stopped event is within the visible region.
+                       Best used in combination with other options. For
+                       example: 'usevisible,usetab,uselast'
```

Comment:
See `:help 'switchbuf'`:

```
	   useopen	If included, jump to the first open window in the
			current tab page that contains the specified buffer
			(if there is one).  Otherwise: Do not examine other
			windows.
	   usetab	Like "useopen", but also consider windows in other tab
			pages.

	   uselast	If included, jump to the previously used window [..]
```